setup.py

```diff
@@ -1,13 +1,9 @@
-import re
 from setuptools import setup
 
 description = 'Version-bump your software with a single command!'
 
-long_description = re.sub(
-  "\`(.*)\<#.*\>\`\_",
-  r"\1",
-  str(open('README.md', 'rb').read()).replace(description, '')
-)
+with open('README.md', 'UTF-8') as readme:
```

Comment:
This breaks on Python 3 with `ValueError: invalid mode: 'UTF-8'`, see the Travis run.